### PR TITLE
David

### PR DESCRIPTION
**Pull request by davyuu**

### CODE REVIEW

File: util/commands.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 var twilio = require('twilio');
 var fs = require('fs');
+var csv = require('fast-csv');
```
Comment (davyuu) [inline util/commands.js]: might not need

File: util/commands.js [hunk 2]
```diff
@@ -293,15 +294,141 @@ function sendTextMessage(message, phoneNumber) {
...
+    console.log('getting current event', param);
+    resp = new twilio.TwimlResponse();
+
+    var obj = getScheduleJson();
+    var day = getDay();
+    var time = getTime();
+
+    //TESTING
+    day = "SATURDAY";
+    time = "16:00";
```
Comment (jstarw) [inline util/commands.js]: if it works, you can remove these

File: util/commands.js [hunk 3]
```diff
@@ -293,15 +294,141 @@ function sendTextMessage(message, phoneNumber) {
...
+    console.log('getting next event', param);
+    resp = new twilio.TwimlResponse();
+
+    var obj = getScheduleJson();
+    var day = getDay();
+    var time = getNextTime();
+
+    //TESTING
+    day = "SATURDAY";
+    time = "16:00";
```
Comment (jstarw) [inline util/commands.js]: same here

**Comment by jstarw** [comment]
Added some comments. Merge whenever those are cleared.

**Pull request merged**